### DISCUSSION TITLE
[@types/js-cookie] CookieReadConverter return type

### DISCUSSION THREAD

**Discussion by khoait** [DefinitelyTyped/DefinitelyTyped#59418] [Issues with a @types package]
Hi,

I'm using @types/js-cookie v3.0.1. I'm trying to implement CookieReadConverter to return a custom type. But the definition is only returning string.

should it be?
`type CookieReadConverter<T extends object> = (value: string, name: string) => string | T;`

Thanks!

[a comment by an automated account is not shown]

**Reply by BendingBender** [reply]
See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59944
